controllers/auth.js

```diff
@@ -41,15 +32,84 @@ exports.superadminLogin = asyncHandler(async (req, res, next) => {
   const token = await superadmin.getSignedJwtToken();
   return res.status(200).json({ success: true, data: token });
 });
+// @forgot password superadmin
+exports.forgotSuperadminPassword = asyncHandler(async (req, res, next) => {
+  const superadmin = await SuperAdmin.findOne({ username: req.body.username });
 
+  if (!superadmin) {
+    return next(new ErrorResponse("There is superadmin with username", 404));
```

Comment:
You have a typo here. It should have been `There is no superadmin with this username`